jaxlib/mosaic/gpu/custom_call.cc

```diff
@@ -38,6 +38,7 @@ limitations under the License.
 #include "absl/status/status.h"
 #include "absl/status/statusor.h"
 #include "absl/strings/str_cat.h"
+#include "absl/strings/str_format.h"
```

Comment:
Please add a dependency on `absl:str_format` in the `BUILD` file. Without it our internal builds aren't working properly

Reply:
I added `"@com_google_absl//absl/strings:str_format"`.